include/exadg/operators/operator_base.cpp

```diff
@@ -849,6 +866,19 @@ OperatorBase<dim, Number, n_components>::reinit_cell(IntegratorCell &   integrat
                                                      unsigned int const cell) const
 {
   integrator.reinit(cell);
+
+  reinit_cell_additional(integrator, cell);
+}
+
+template<int dim, typename Number, int n_components>
+void
+OperatorBase<dim, Number, n_components>::reinit_cell_additional(IntegratorCell &   integrator,
+                                                                unsigned int const cell) const
+{
+  (void)integrator;
+  (void)cell;
+
+  // overwrite this function in derived classes if additional initialization is necessary
```

Comment:
Regarding the name, should we use the common C++ idiom,
```suggestion
  // override this function in derived classes if additional initialization is necessary
```